packages/sdks/typescript/rollup.config.mjs

```diff
@@ -0,0 +1,25 @@
+import typescript from '@rollup/plugin-typescript'
+import { dts } from 'rollup-plugin-dts'
+
+const config = [
+  {
+    input: 'src/index.ts',
+    output: {
+      file: 'dist/index.js',
+      format: 'es',
+      sourcemap: true,
+    },
+    plugins: [
+      typescript({
+        exclude: ['**/__tests__', '**/*.test.ts'],
+      }),
+    ],
+  },
+  {
+    input: 'src/index.ts',
+    output: [{ file: 'dist/index.d.ts', format: 'es' }],
+    plugins: [dts()],
```

Comment:
Use alias

Reply:
I don't anticipate this project getting too large so I'm just gonna skip aliases altogether